server/api/order.js

```diff
@@ -0,0 +1,39 @@
+const router = require('express').Router()
+const { Order, Costume } = require('../db')
+
+router.get('/', async(req, res, next) => {
+    try {
+        const orders = await Order.findAll()
+        res.send(orders)
+    } catch(err) {
+        next(err)
+    }
+})
+//currently doing findOne since user should only have one order that is not paid
+router.get('/userCart', async(req,res,next) => {
+    try{
+        const userId = req.user.id
+        const userCart = await Order.findOne({
+            where: {
+                userId,
+                isPaid: false,
+            }, 
+            include: [Costume]
+        })
+        res.send(userCart)
+    }catch(err) {
+        next(err)
+    }
+})
+
+router.get('/:id', async(req, res, next) => {
+    try {
+        const order = await Order.findByPk(req.params.id, { include: [Costume] })
+        res.send(order)
```

Comment:
Any user can view any order if they have the order id? Aren't there some privacy concerns with that?